test/unit/tokenizer_unit_test.rb

```diff
@@ -34,15 +34,15 @@ def test_calculate_line_numbers_per_token_with_profiling
 
   def tokenize(source)
     tokenizer = Liquid::Tokenizer.new(source)
-    tokens = []
-    while (t = tokenizer.shift)
+    tokens    = []
+    while (t  = tokenizer.shift)
```

Comment:
This is about the only one that doesn't seem to make sense.

Reply:
Ah! I missed that! :stuck_out_tongue: 
It should be something like:
```ruby
    tokenizer = Liquid::Tokenizer.new(source)
    tokens    = []

    while (t = tokenizer.shift)
```
I can't push to this branch directly,.. so its your call..